pdbufr/bufr_structure.py

```diff
@@ -14,6 +14,7 @@
 import eccodes  # type: ignore
 import numpy as np  # type: ignore
 
+from shapely.geometry import Point
```

Comment:
Please, make _shapely_ optional (not sure what would be the best way).

Reply:
I decided to use again your try ... except block to use geometry as a normal list in case of HAS_GEOPANDAS=False